packages/libllvm/tools-clang-lib-Driver-ToolChains-Linux.cpp.patch

```diff
@@ -1,74 +1,28 @@
 --- tools/clang/lib/Driver/ToolChains/Linux.cpp.orig	2019-12-21 22:30:03.676720096 +0000
 +++ ./tools/clang/lib/Driver/ToolChains/Linux.cpp	2019-12-21 22:34:42.941719383 +0000
-@@ -316,6 +316,41 @@
+@@ -403,7 +403,7 @@ Linux::Linux(const Driver &D, const llvm::Triple &Triple, const ArgList &Args)
+   if (Triple.getVendor() == llvm::Triple::OpenEmbedded &&
+       Triple.isArch64Bit())
+     addPathIfExists(D, SysRoot + "/usr/" + OSLibDir, Paths);
+-  else
++  else if (!IsAndroid)
+     addPathIfExists(D, SysRoot + "/usr/lib/../" + OSLibDir, Paths);
```

Comment:
This adds `-L $TERMUX_PREFIX/usr/lib/../lib` to the linker search path, which can't be listed first for cross-compiling.